src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/duration.pipe.ts

```diff
@@ -13,6 +13,9 @@ export class DurationPipe implements PipeTransform {
    * @return {string}         The phrase describing the the amount of time
    */
   transform(seconds: number): string {
+    if (seconds === null || seconds <= 0) {
```

Comment:
It seems the tests need to be adapted (make check failure):
```
  ● DurationPipe › transforms seconds into a human readable duration

    expect(received).toBe(expected) // Object.is equality

    Expected: "1 second"
    Received: ""

       9 |
      10 |   it('transforms seconds into a human readable duration', () => {
    > 11 |     expect(pipe.transform(0)).toBe('1 second');
         |                               ^
      12 |     expect(pipe.transform(6)).toBe('6 seconds');
      13 |     expect(pipe.transform(60)).toBe('1 minute');
      14 |     expect(pipe.transform(600)).toBe('10 minutes');
```

Reply:
done.